plugins/mcreator-localization/lang/texts.properties

```diff
@@ -171,6 +171,14 @@ trigger.entity_pickupxp=Entity picks up experience
 trigger.player_useitem_start=Player uses item
 trigger.player_useitem_stop=Player stops using item
 trigger.player_useitem_finish=Player finishes using item
+trigger.entity_sets_attack_target=Entity sets its attack target
+trigger.entity_grief=Entity about to grief
+trigger.player_xp_change=Player xp amount changes
+trigger.player_xp_level_change=Player xp level changes

```

Comment:
Noticed xp is lowercase, in MCreator, XP is always written uppercase. Fix this for newly added triggers.